### PR TITLE
add i18n and edit model

### PR DESCRIPTION
**Pull request by kienpham2004**
Peek 2020-12-04 10-23

## Related Tickets:
- [#32405](https://edu-redmine.sun-asterisk.vn/issues/32405)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by kienpham2004** [comment]
ready

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -40,6 +44,12 @@
                     <ul class="navbar-nav ml-auto">
                         <!-- Authentication Links -->
                         @guest
+                            <li class="nav-item">
+                                <a class="nav-link" href="{!! route('home.lang', ['vi']) !!}">Tiếng Việt</a>
```
Comment (TDM1911) [inline resources/views/layouts/app.blade.php]:
```suggestion
                                <a class="nav-link" href="{{ route('home.lang', ['vi']) }}">Tiếng Việt</a>
```

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -40,6 +44,12 @@
...
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="{!! route('home.lang', ['en']) !!}">English</a>
```
Comment (TDM1911) [inline resources/views/layouts/app.blade.php]:
```suggestion
                                <a class="nav-link" href="{{ route('home.lang', ['en']) }}">English</a>
```

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -57,6 +67,12 @@
                                 <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
                                     <form id="logout-form" action="{{ route('logout') }}" method="POST">
                                         @csrf
+                                        <li class="nav-item">
+                                            <a class="nav-link" href="{!! route('home.lang', ['vi']) !!}">Tiếng Việt</a>
```
Comment (TDM1911) [inline resources/views/layouts/app.blade.php]:
```suggestion
                                            <a class="nav-link" href="{{ route('home.lang', ['vi']) }}">Tiếng Việt</a>
```

File: resources/views/layouts/app.blade.php [hunk 4]
```diff
@@ -57,6 +67,12 @@
...
+                                        </li>
+                                        <li class="nav-item">
+                                            <a class="nav-link" href="{!! route('home.lang', ['en']) !!}">English</a>
```
Comment (TDM1911) [inline resources/views/layouts/app.blade.php]:
```suggestion
                                            <a class="nav-link" href="{{ route('home.lang', ['en']) }}">English</a>
```

**Review by TDM1911 — COMMENTED** [review]
Pull **n1** nghĩa là gì?

**Review by huyenttm-0901 — COMMENTED** [review]
sửa lại title pull, commented

**Comment by kienpham2004** [comment]
ready

**Pull request merged**